ui/src/components/HelpAccordion.tsx

```diff
@@ -83,10 +84,16 @@ const StyledBox = styled.div`
 interface HelpDrawerProps {
   title: string;
   body: string;
+  login: boolean;
 }
 
-export function HelpAccordion({ title, body }: HelpDrawerProps): JSX.Element {
+export function HelpAccordion({
+  title,
+  body,
+  login,
```

Comment:
let's make this false by default so we don't have to keep repeating `login={false}`

```suggestion
  login = false,
```

Reply:
okay, made login into an optional field and set it to false by default